backend/src/main/resources/db/migration/V1__create_events_and_user.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE events
+(
+    id           BIGINT GENERATED BY DEFAULT AS IDENTITY NOT NULL,
+    name         VARCHAR(250),
+    start_date   TIMESTAMP WITHOUT TIME ZONE,
+    end_date     TIMESTAMP WITHOUT TIME ZONE,
+    description  VARCHAR(2000),
+    organizer_id BIGINT,
+    CONSTRAINT pk_events PRIMARY KEY (id)
+);
+
+CREATE TABLE users
+(
+    id    BIGINT GENERATED BY DEFAULT AS IDENTITY NOT NULL,
+    login VARCHAR(255)                            NOT NULL,
```

Comment:
Wszystkie pola tekstowe mają 255 długości, co nie powinno mieć miejsca. login i type to varchary odpowiednio 128 i 32, a hash i salt to stringi stałej długości odpowiednio 64 i 32.

Reply:
Nie na tym polegało zadanie, ale poprawiłem

Reply:
Ale tak było przed twoimi zmianami.